### PR TITLE
[#2633] Added option to backfill for start and end dates in YYYYMMDD …

### LINKED ISSUE #2633: Add support for start and end dates in YYYYMMDD format to backfill command
Just that, I've added that option to backfill, my brain can't stand working with epoch ms dates....

### PR DESCRIPTION
**Pull request by mmdiego**
Closes #2633

### CODE REVIEW

File: commands/backfill.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ var tb = require('timebucket')
   , crypto = require('crypto')
   , objectifySelector = require('../lib/objectify-selector')
   , collectionService = require('../lib/services/collection-service')
+  , debug = require('../lib/debug')
```
Comment (DeviaVir) [inline commands/backfill.js]: not used?

File: commands/backfill.js [hunk 2]
```diff
@@ -12,6 +14,8 @@ module.exports = function (program, conf) {
     .option('-d, --days <days>', 'number of days to acquire (default: ' + conf.days + ')', Number, conf.days)
     .option('--start <unix_in_ms>', 'lower bound as unix time in ms', Number, -1)
     .option('--end <unix_in_ms>', 'upper bound as unix time in ms', Number, -1)
+    .option('--start_date <datetime>', 'start date in format ("YYYYMMDDhhmm")', Number, -1)
+    .option('--end_date <datetime>', 'end date in format ("YYYYMMDDhhmm")', Number, -1)
```
Comment (DeviaVir) [inline commands/backfill.js]: can we support both unix timestamp and datetime stamp for the same parameter?

File: commands/backfill.js (hunk 1 above)
Comment (mmdiego) [reply commands/backfill.js]: oh no, sorry, i'm using it locally but I didn't remove it from the commit, can you do it during the merge?

File: commands/backfill.js (hunk 2 above)
Comment (mmdiego) [reply commands/backfill.js]: i have no idea, let me check moment docs to see if there is a way to diffrentiate them

**Comment by mmdiego** [comment]
What about now? Hope it works ok.. :)

**Review by DeviaVir — APPROVED** [review]
:+1: LGTM, thanks for the contribution

**Pull request merged**